envs/picasso/readme

```diff
@@ -0,0 +1,13 @@
+This image is currently built with a manual trigger from the github actions page.
```

Comment:
Seems a bit odd having a readme update shoved in with the rest of the change. Why is this here now?

Reply:
Our internal environment commands broke so we switched to testing in the Singularity container. That needed a change to the way we run the container; setting a couple of environment variables so git can work. This readme records that so other people can see how to run the container if the environment breaks again.

Reply:
Still not happy this has been shoved in with an unrelated changeset- it fails the 'also' test. However, as it's a readme I'll let it go.